hera_mc/cm_active.py

```diff
@@ -38,11 +38,7 @@ def get_active(
     active : ActiveData object
         ActiveData objects with loading parameters as specified.
     """
-    if testing:
-        db = mc.connect_to_mc_testing_db()
-    else:  # pragma: no cover
-        db = mc.connect_to_mc_db(None)
-    with db.sessionmaker() as session:
+    with mc.MCSessionWrapper(None, testing=testing) as session:
```

Comment:
I really want us to pass the session as a named parameter rather than positionally. It's easier to read and is robust against argument order changes.

```suggestion
    with mc.MCSessionWrapper(session=None, testing=testing) as session:
```